spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/732/posts' do
+    it 'returns a 200 OK status' do
+      get '/users/732/posts'
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the index template' do
+      get '/users/732/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'renders the show template' do
+      get '/users/732/posts/2'
+      expect(response).to render_template(:show)
+    end
+
+    it 'contains the post ID in the URL' do
+      get '/users/732/posts/2'
+      expect(request.fullpath).to include('/users/732/posts/2')
+    end
```

Comment:
- [ ] Please consider writing the GET/show testing method in a different block using a new `describe` method to separate the tests and make it easier to identify. 🙏

_Kindly do the same for the GET/show testing method in the users_spec as well._